pkg/feature/templates/servicemesh/authorino/base/operator-cluster-wide-no-tls.tmpl

```diff
@@ -0,0 +1,14 @@
+apiVersion: operator.authorino.kuadrant.io/v1beta1
+kind: Authorino
+metadata:
+  name: {{ .AuthorinoConfigs.Name }}
+  namespace: {{ .Auth.Namespace }}
+spec:
+  authConfigLabelSelectors: authorino/topic=odh
```

Comment:
I applied the suggestion...

Although, given both the Istio AutorizationPolicy and the Authorino AuthConfigs are quite specific to the details of KServe, I wounder if we may want to set a group other than `default`. Not sure if it may be useful some day.

Reply:
They're now, but with more components coming to the picture and potential per-project authorization rules this can be helpful. I was actually thinking to prefix it with `sec.` so it will indicate `security` category. 

`sec.opendatahub.io/authorization-group: default` 

WDYT @israel-hdez @aslakknutsen

Reply:
I'm OK with the additional prefix, but I'm not sure I like shortening it.

Reply:
Good point, aligned it with other labels we use across the tools, e.g. `security.openshift.io/scc.podSecurityLabelSync`